02-apply_maxwell_filter.py

```diff
@@ -2,24 +2,24 @@
 ===================================
 03. Maxwell filter using MNE-python
 ===================================
-
-XXX: this needs update: congig.mf_reference_run, config.mf_st_duration
-config.mf_cal_fname, config.mf_ctc_fname
-
-The data are Maxwell filtered using tSSS and movement compensation.
-
+  
+The data are Maxwell filtered using SSS or tSSS (if config.mf_st_duration is not None)
+and movement compensation.
 Using tSSS with a short duration can be used as an alternative to highpass
 filtering. Here we will use the default (10 sec) and a short window (1 sec).
```

Comment:
I don't think this is true here anymore.

Reply:
can you clean this one up too?

Reply:
Yes, if config.mf_st_duration ~= None it runs tSSS